src/v2/view-builder/views/password/ReEnrollAuthenticatorWarningPasswordView.js

```diff
@@ -0,0 +1,38 @@
+import { loc } from 'okta';
+import BaseFooter from '../../internals/BaseFooter';
+import EnrollAuthenticatorPasswordView from './EnrollAuthenticatorPasswordView';
+
+const Body = EnrollAuthenticatorPasswordView.prototype.Body.extend({
+  className: 'password-authenticator',
+  title () {
+    const daysToExpiry = this.options.appState.get('currentAuthenticator').settings.daysToExpiry;
+    return `${loc('oie.password.expiry.warning.title', 'login')}
+      ${daysToExpiry} ${loc('oie.password.expiry.warning.time','login')}`;
+  },
+
+  save () {
+    return loc('oie.password.expired.primaryButton', 'login');
+  },
+
+  getPasswordPolicy () { 
+    return this.options.appState.get('currentAuthenticator').settings;
+  },
+});
+
+const Footer = BaseFooter.extend({
+  links () {
+    const links = [];
+    if (this.options.appState.hasRemediationObject('skip')) {
+      const skipConfig = this.options.appState.get('remediations').find(remediation => remediation.name === 'skip');
```

Comment:
Should this go elsewhere? like a derived property on appState or something?

Reply:
use `actionPath: skip` instead of `href` at line 31